### DISCUSSION TITLE
Allow timeline zooming in AnimationPlayer without needing to hover over the time bar.

### DISCUSSION THREAD

**Discussion by yosimba2000** [godotengine/godot-proposals#10399] [Editor]
If you want to zoom in on the timeline in AnimationPlayer using scrollwheel, your cursor must hover over the timebar on top, or the slider on the bottom right.

Please make it so it works when hovering in the main timeline area as well.

![Untitled](https://github.com/user-attachments/assets/cd75d8dc-dd14-4672-88cc-492a8b7d7061)

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/6681.

The main area (the one highlighted by "cant zoom" on the image) is handled by a different class than the timeline, so the zooming functionality would need to be implemented twice (unless there's a way to forward it to the timeline). I recall there's one-way communication between the timeline and the main area, but I don't remember in which direction it goes.

Either way, if you're interested in implementing this, here's the relevant code:

https://github.com/godotengine/godot/blob/739019e4e4a6e4763e37adfd9883a1c85d5f6249/editor/animation_track_editor.cpp#L1793-L1805

**Reply by yosimba2000** [reply]
Thanks for the info! I made a comment on that thread.
